src/main/flight/pid.c

```diff
@@ -740,6 +741,48 @@ float pidGetAirmodeThrottleOffset(void)
 }
 #endif
 
+// when new Rx data is received, update the maxDeflectionAbs value, so we don't do this every PID loop
+static float maxDeflectionAbs = 0.0f;
+float sendMaxDeflectionAbs(float maxRcDeflectionAbs)
+{
+    maxDeflectionAbs = fmaxf(maxRcDeflectionAbs, mixerGetRcThrottle());
+    return maxDeflectionAbs;
+}
+
+static void disarmOnImpact(const int axis)
```

Comment:
This should not be dependent on the axis.
```suggestion
static void disarmOnImpact(void)
```

Reply:
Thanks, you're right.  Fixed.